services/dub/dub-download.service.js

```diff
@@ -1,11 +1,9 @@
 'use strict'
 
 const Joi = require('joi')
-const BaseJsonService = require('../base-json')
 const { metric } = require('../../lib/text-formatters')
-const {
-  downloadCount: downloadCountColor,
```

Comment:
Sidebar comment: I know it's been mentioned in other threads we want to remove the import renames like this, which makes sense, but there's one formatter (I think it's `version`?) where the function name is identical to the most frequently used parameter name. What do folks think about tweaking some of the format function names to more explicitly reference the formatting aspect? 

Presumably folks were renaming those imports for a similar reason?

Reply:
Yes, agreed.

The history sort of goes the other way. When those helper functions were pulled out into modules, some of them were given new names (and then the renames were placed at the top of 7k-line `server.js`). When I split up `server.js` I moved the route handlers intact so the renames got copied along with them.

A lot of the new names were good standalone, though a few of them were styled in a way that made them difficult to understand without the context of their module. I suppose scoping them under their module is a reasonable stylistic choice, though it seems hard to maintain in a project like this.

Reply:
Gotcha. It's the `version` color formatter function that always makes me feel guilty since I almost always have to rename the import.